apps/editing-toolkit/editing-toolkit-plugin/maps-provider/index.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Map provider
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE;
+
+/**
+ * Adds a global variable containing the map provider to the window object
+ *
+ * @return void
+ */
+function set_map_provider() {
+	$map_provider = apply_filters( 'wpcom_map_block_map_provider', 'mapbox' );
+
+	wp_register_script( 'map-block-map-provider', '', array(), '1.0', true );
+	wp_localize_script( 'map-block-map-provider', 'map_block_map_provider', $map_provider );
```

Comment:
I believe `$map_provider` should be an array here:
https://developer.wordpress.org/reference/functions/wp_localize_script/

You will get PHP notices when this runs.

Reply:
Thanks for catching this! `jetpack-mu-wpcom` seems like a much better place for this code to live. 

I'll create a new PR that removes it here and adds it in `jetpack-mu-wpcom` (as an array).

Reply:
I have draft PR's ready for this, and I've updated the Mapkit PR to accommodate for getting an object back instead of a string

Remove from ETK: https://github.com/Automattic/wp-calypso/pull/76230
Add to jetpack-mu-wpcom: https://github.com/Automattic/jetpack/pull/30287
Mapkit: https://github.com/Automattic/jetpack/pull/29098/commits/0cfebc777f084d5c6e5a280ffcf75d97ac47c4c6

I'll get these approved by my team tomorrow. Thanks for the help today! 🙏